modules/components/src/DataTable/TableToolbar/TableToolbar.js

```diff
@@ -120,35 +131,66 @@ const TableToolbar = ({
               {columnDropdownText}
             </DropDown>
           ))}
-        {allowTSVExport && (
-          <div className="buttonWrapper">
-            <button
-              style={{
-                display: 'flex',
-                cursor: 'pointer',
-                whiteSpace: 'nowrap',
-                minHeight: 16,
-              }}
-              onClick={() => {
-                exporter(
-                  transformParams({
-                    url: downloadUrl,
-                    files: [
-                      {
-                        fileName: exportTSVFilename,
-                        fileType: 'tsv',
-                        sqon,
-                        index: type,
-                        columns,
-                      },
-                    ],
-                  }),
-                );
-              }}
-            >
-              {exportTSVText}
-            </button>
-          </div>
+
+        {multipleExporters ? ( // check if we're given more than one custom exporter
+          <DropDown
+            aria-label={`Download options`}
+            itemToString={(i) => i.exLabel}
+            items={exporterArray}
+            onChange={({ exColumns, exLabel, exFunction }) =>
```

Comment:
For these variables it is unclear what `ex` means. I assumed `extra` but thats not aligned with the code here, so probably means something else I am missing. As a general rule, don't use abbreviations in variable names. Long variable names are no issue.

Reply:
You're 100% right, and I'm normally of the same policy. 
Guess working late at night is not doing wonders for my code quality these days, uh?
😆 💀 

Changes committed!